contents/basepages/aboutus/index.en.mdx

```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
 ---
 Sorry, now translating...
 
+title: Profile of Kikuo Tsutsumi
+date: 2023-01-19
+author: TSUTSUMI Kikuo
+slug: /aboutme
+images: 
+  - kikuo-profile-1-circle.png
+  - AliceNail-Testimonial.jpg
+  - kikuo-suffering.jpg
+showTOC: false
+---
+## self-introduction
+
+<ImageBox alt="TSUTSUMI Kikuo" float="right" 
+       image={0} bgColor="transparent"
+       style={{borderColor: "transparent"}} />
+
+Google Maps Acquisition Expert for Small Businesses
+- Learned internet marketing from sluggish sales of side business
+- Previously worked as an internet and web-related programmer
+- Majored in business administration, software engineering and artificial intelligence at university
+- Currently, we are spreading the "Google Maps method of attracting customers" that can attract customers for zero yen to small shop owners.
+
+### the beginning
+
+When I got married, I realized that I had no future as a company employee.
+I decided to start a side business of making handmade products.
+
+Since I was a child, I have been making things, drawing pictures, taking photographs, making small films, and performing on stage in theater.
+Because I love creating things.
+
+### You never know until you try
+
+<ImageBox alt="Suffering" float="left" image={2}
+	  style={{borderWidth: "1px"}} >
+<footnote>When I was worried about not selling</footnote>
+</ImageBox>
+
+However, to my surprise, it did not sell at all. In the first place, you can't even see the product page.
+
+I was very particular about it and spent a lot of time making it, but I was worried about why no one was interested.
+I was selfish and even felt resentment, thinking that if people could hold it in their hands, look at it, and use it, they would understand how good it is.
+
+But I can't afford to open a store.
+I had no choice but to sell it online.
+Now you can't even pick it up.
+
+### Try to change your mind
+- "A change in perspective is worth 80 IQ points." -- Alan Key (A father of personal computer.)<br/>
+  "A change of perspective is worth 80 points of IQ." -- Alan Kay, father of the personal computer
```

Comment:
It's the same as the previous line, so please delete this line. Please also delete the <br/> in the previous line.

Reply:
done